src/Stratis.Bitcoin.Features.PoA/Voting/Poll.cs

```diff
@@ -32,6 +32,9 @@ public Poll()
         /// </summary>
         public bool IsPending => this.PollVotedInFavorBlockData == null;
 
+        /// <summary><c>true</c> if poll has expired; <c>false</c> otherwise.</summary>
+        public bool IsExpired => this.PollVotedInFavorBlockData?.Height == 0;
```

Comment:
I feel that this can be confusing as just because the poll has not had its poll finished, it shouldn't be regarded "Expired" ?

Reply:
Sorry, I know this PR is draft but just thought about this now.

Reply:
Fixed.